data_management/autopreprocess_testing/preprocess_directory.py

```diff
@@ -19,9 +19,10 @@
 target_dir = os.getcwd() + '/processed_test_images/'
 
 # Get list of all image paths
-jpg_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
-png_files = glob.glob(source_dir + "/**/*.png", recursive=True)
-img_files = jpg_files + png_files
+supported_file_extensions = ["jpg", "jpeg", "png", "tif", "tiff"]
```

Comment:
This is very helpful work, Jay. This refactored code should make it easier with preprocessing various types of data.

Reply:
Changing the code to be DRY using a for loop is really helpful. Thank you.